src/components/search-bar/search-result-header/SearchResultHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { useText } from "../../../core/utils/text";
+
+export interface SearchResultHeaderProps {
+  hitcount?: string;
+  q?: string;
```

Comment:
What are you thoughts about the props being required/optional?
The way the rendering in the component is constructed it does not handle if one of them are empty/falsy